.github/workflows/coder.yaml

```diff
@@ -102,6 +102,15 @@ jobs:
         with:
           version: v1.46.0
 
+  check-enterprise-imports:
+    name: check/enterprise-imports
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Check imports of enterprise code
+        run: ./scripts/check_enterprise_imports.sh
+
```

Comment:
Would it be OK to just let the lint step handle this?

Reply:
I don't think the lint Action calls `make lint/go`, it directly calls golangci-lint